building/devices/zynqmp.rst

```diff
@@ -1,23 +1,42 @@
 .. _zynqmp_zcu102:
```

Comment:
You can and should probably also change the name of this, since it's now used more generically. I'd recommend just using `.. _zynqmp` or `.. _xilinx` or `.. _xilinx_devices` (see examples in the other rst files).